atomdb/datasets/slater/run.py

```diff
@@ -39,6 +40,65 @@
 # DATAPATH = os.path.abspath(DATAPATH._paths[0])
 
 
+@dataclass
```

Comment:
You shouldn't have to use dataclass here, you should be using this instead: https://www.pytables.org/usersguide/libref/declarative_classes.html#isdescriptionclassdescr